src/Sciences.API/Controllers/SciencesController.cs

```diff
@@ -14,19 +15,19 @@ public async Task<IActionResult> GetSciences()
     }
 
     [HttpPost]
-    public async Task<IActionResult> AddScience()
+    public async Task<IActionResult> AddScience(CreateScienceModel model)
     {
         return Ok();
     }
 
     [HttpGet("{science_id}")]
-    public async Task<IActionResult> GetScienceById(Guid science_id)
+    public async Task<IActionResult> GetScienceById(Guid scienceId)
```

Comment:
[HttpGet("{science_id}")] bundagini ham togrlab qoying

Reply:
To'g'rilandi👍